server/integration/infinispan/src/main/java/org/jboss/as/clustering/infinispan/subsystem/InfinispanExtension.java

```diff
@@ -81,9 +81,6 @@ public void initialize(ExtensionContext context) {
 
         subsystem.registerSubsystemModel(new InfinispanSubsystemRootResource(resolvePathHandler, context.isRuntimeOnlyRegistrationValid()));
         subsystem.registerXMLElementWriter(new InfinispanSubsystemXMLWriter());
-        if (context.isRegisterTransformers()) {
```

Comment:
Yes, before 1.4 the schema didn't allow expressions in place of values (e.g. ${...}). The transformers were put in place to be able to handle that when translating configurations between the various versions (e.g. newer domain controller with older hosts)